src/sentry/conf/server.py

```diff
@@ -1187,20 +1187,16 @@ def SOCIAL_AUTH_DEFAULT_USERNAME():
     # Enable SAML2 based SSO functionality. getsentry/sentry-auth-saml2 plugin
     # must be installed to use this functionality.
     "organizations:sso-saml2": True,
-    # Enable the server-side sampling feature (backend + relay)
+    # Enable the server-side sampling (backend + relay)
```

Comment:
Is this flag still necessary? Given that it is [only set when `dynamic-sampling` is _not_ set](https://github.com/getsentry/sentry/pull/40388/files#diff-5e30755537fcfd5687d4399272065e49affcc12a4cf7a0bc377cff5234b8a08aR196), could we consolidate this flag with `dynamic-sampling-deprecated` now?

Reply:
We considered it, but it would require more changes in the code that will be removed after November.

So to make things as time-efficient as possible, we decided on this approach and we'll do a cleanup when removing the deprecated code after the end of November.